Source/relay/TourGuide/Quests/Level 13.ash

```diff
@@ -247,32 +247,32 @@ boolean generateTowerFamiliarWeightMethod(string [int] how, string [int] immedia
     TFWMInternalModifier best_familiar_equipment;
     foreach key in weight_modifiers
     {
-        TFWMInternalModifier modifier = weight_modifiers[key];
-        if (modifier.have && modifier.from_familiar_equipment)
+        TFWMInternalModifier weight_modifier = weight_modifiers[key];
```

Comment:
realistically i'd rather all these be more like this where it names clearly what it is... but i don't care enough to block a breaking mergefix

Reply:
I'll go back and clean up in a follow-up 👍